core/src/rpc.rs

```diff
@@ -155,7 +154,7 @@ impl JsonRpcRequestProcessor {
         &self,
         pubkey: Result<Pubkey>,
         commitment: Option<CommitmentConfig>,
-    ) -> RpcResponse<Option<RpcAccount>> {
+    ) -> Result<RpcResponse<Option<RpcAccount>>> {
```

Comment:
The rpc macro requires the `returns` attribute on any return value not named `Result`.  Also, the `Response` type appears to conflict with some internal type named `Response` so we alias it.